pkg/apis/core/types.go

```diff
@@ -1564,6 +1584,14 @@ const (
 	MountPropagationBidirectional MountPropagationMode = "Bidirectional"
 )
 
+// VolumeDevice describes a mapping of a raw block device within a container.
+type VolumeDevice struct {
+	// Required: name must match the name of a persistentVolumeClaim volume in the pod
```

Comment:
Remove `// Required: ` and replace with `// name must match the name of a persistentVolumeClaim in the pod`.  These fields are already marked as required in open api and no further comment is necessary.  This and field below.